Nighttime-Operations/Observatory/DIMM/index.rst

```diff
@@ -4,11 +4,52 @@
 DIMM
 ###############
 
-This page contains some common calibrations procedures.
+The differential image motion monitor (DIMM) is a standard instrument to measure astronomical "seeing", or atmospheric turbulence strength. 
```

Comment:
consider: 
.... "seeing" which is a metric for the amount of atmospheric turbulence effecting image quality.

Reply:
Sounds better.